app/lib/common/utils/constants.dart

```diff
@@ -35,6 +36,7 @@ const autoGuestLogin = Env.autoLoginAsGuest;
 const inCI = Env.isCI;
 const isDemo = Env.isDemo;
 const isNightly = Env.isNightly;
+const includeChatShowcase = isNightly || isDevBuild;
```

Comment:
Ohh I see that's only enabled for dev/nightly builds. Maybe we want to have it generally visible for end-user also ? I guess that can be left for future discussion.